### DISCUSSION TITLE
Something went wrong Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by Zi-Chung** [GaiZhenbiao/ChuanhuChatGPT#758] [General]
请问这个问题要怎么解决呢?
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/133855717/f5a92964-3358-456a-8a3d-b8927cea2ea1)
代理配置也按照up的方法改了,网络代理也能够使用
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/133855717/9e8616c0-f800-469a-a3fd-3fbe291b80a2)
求大佬解答!!!谢谢

**Comment by GaiZhenbiao** [comment]
开启Clash的TUN模式试试？

**Reply by Zi-Chung** [reply]
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/133855717/c77323a4-866d-4a8a-a2a0-8435196a030f)
已经开了啊,可是还是显示Something went wrong Expecting value: line 1 column 1 (char 0)

**Comment by Keldos-Li** [comment]
参考：
- #328

**Reply by Zi-Chung** [reply]
这个是要把Clash里除了TUN Mode和Service Mode旁边的小地球点亮以外其他都不开吗?可是我不开其他的网络代理就用不了了,然后Service Mode旁边的地球也点不亮,不知道为什么.望解答,谢谢!
